### PR TITLE
AOM table: change column order. Closes #932

### LINKED ISSUE #932: Tableau des AOMs - relocaliser le temps réel
Il semble plus facile pour manipuler la chose efficacement que la colonne "temps réel" soit juste après la colonne "à jour". C'est plus simple à comprendre d'avoir toutes les cases oui/non regroupées et les "infos statistiques" à droite.

### PR DESCRIPTION
**Pull request by Tristramg**

### CODE REVIEW

**Comment by antoine-de** [comment]
it's good for me.

I took the liberty to style of bit the download button:

image

**Comment by Tristramg** [comment]
ohlalala ! such front dev !

File: apps/transport/lib/transport_web/templates/aoms/index.html.eex [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <h2>Situation des AOM pour l’ouverture des données transport en commun</h2>
-
-<table class="table">
+<p><%= link "Télécharger en CSV", to: aoms_path(@conn, :csv) %></p>
```
Comment (antoine-de) [inline apps/transport/lib/transport_web/templates/aoms/index.html.eex]:
```suggestion
<div class="is-centered">
  <a class="button" href="<%= aoms_path(@conn, :csv) %>" role="link">
      <i class="icon icon--download" aria-hidden="true"></i>
      Télécharger en CSV
  </a>
</div>
```

to get 
image

**Comment by antoine-de** [comment]
:sunglasses:

**Pull request merged**